### PR TITLE
Added missing guides

### PR DESCRIPTION
**Pull request by v-troywarren**
Added the following guides.
Identity Governance
Secure your cloud apps with Single Sign On (SSO)
Migrate from ADFS to Azure AD

Updated
Plan Passwordless Deployment

### CODE REVIEW

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 1]
```diff
@@ -83,12 +83,13 @@ Advanced deployment guides in the admin center require authentication to a Micro
...
+||[Configure multi-factor authentication (MFA) guide](https://aka.ms/MFASetupGuide)| **Configure multi-factor authentication (MFA) guide** for customers with Azure P1 or Azure P2, we provide customizable Conditional Access templates that include the most common and least intrusive security standards. When Azure licensing isn’t available, we provide a one-click solution to enable Security Defaults, a baseline protection policy for all users, or we provide steps to enable legacy (per-user) MFA.|
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
||[Configure multi-factor authentication (MFA) guide](https://aka.ms/MFASetupGuide)| The **Configure multi-factor authentication (MFA) guide** for customers with Azure P1 or Azure P2 provides customizable Conditional Access templates that include the most common and least intrusive security standards. When Azure licensing isn’t available, we provide a one-click solution to enable Security Defaults, a baseline protection policy for all users, or we provide steps to enable legacy (per-user) MFA.|
```

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 2]
```diff
@@ -134,3 +136,4 @@ Advanced deployment guides in the admin center require authentication to a Micro
...
 ||[SDS Rollover setup guide](https://aka.ms/sdsrolloversetupguide)|The **SDS Rollover setup guide** provides the steps to help your organization sync student information data to Azure Active Directory and Office 365. This guide streamlines the term lifecycle management process by creating Office 365 Groups for Exchange Online and SharePoint Online, class teams for Microsoft Teams and OneNote, as well as Intune for Education, and rostering and single sign-on integration for third-party apps. You'll perform end-of-year closeout, tenant cleanup and archive, new school year preparation, and new school year launch. Then you can create new profiles using the sync deployment method that suits your organization.|
+||[Migrate from ADFS to Azure AD](https://aka.ms/MigrateADFStoAzure)|In **Migrate from ADFS to Azure AD** we offer custom guidance for migrating from ADFS to Azure AD. You'll first answer a few questions about your Active Directory Federation Services (AD FS) infrastructure. Then implement either pass-through authentication (PTA) or password hash sync (PHS) to give users a streamlined experience while accessing your org's apps.|
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
||[Migrate from ADFS to Azure AD](https://aka.ms/MigrateADFStoAzure)|In **Migrate from ADFS to Azure AD** we offer custom guidance for migrating from Active Directory Federation Services (AD FS) to Azure AD. You'll first answer a few questions about your AD FS infrastructure. Then implement either pass-through authentication (PTA) or password hash sync (PHS) to give users a streamlined experience while accessing your organization's apps.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-troywarren, kindly commit the suggested changes. Many thanks.

**Comment by v-troywarren** [comment]
@JohanFreelancer9 all suggestions committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by OhanNanen-MS** [comment]
This PR doesn't seem to include recent changes and would overwrite those. Please do not merge as is.

**Pull request closed without merging**